install.js

```diff
@@ -273,6 +267,10 @@ function generatePromptOverrides() {
     var configData = JSON.parse(JSON.stringify(configJson).replace(/true/g, '"y"').replace(/false/g, '"n"'));
     configData.install = 'y';
   }
+
+  const isSessionSecretDefined = USE_CONFIG && configData.sessionSecret;
+  const sessionSecret = (isSessionSecretDefined) ? configData.sessionSecret : crypto.randomBytes(64).toString('hex');
```

Comment:
`isSessionSecretDefined` will end up being the secret itself if defined, so suggest just the one var:
```js
const sessionSecret = USE_CONFIG && configData.sessionSecret ||
  crypto.randomBytes(64).toString('hex');
```